src/unix/stream.c

```diff
@@ -301,7 +301,9 @@ int uv__stream_try_select(uv_stream_t* stream, int* fd) {
   if (fds[1] > max_fd)
     max_fd = fds[1];
 
-  sread_sz = (max_fd + NBBY) / NBBY;
+  sread_sz = (max_fd + 1) / NBBY;
+  if (sread_sz % sizeof(uint32_t) != 0)
+    sread_sz += sizeof(uint32_t) - (sread_sz % sizeof(uint32_t));
```

Comment:
Can I suggest adding `#define ROUND_UP(a, b) ((a) % (b) ? ((a) + (b)) - ((a) % (b)) : (a))` to src/unix/internal.h and then replacing this with `sread_sz = ROUND_UP(max_fd + 1, sizeof(uint32_t))`?

If nothing else, it would remove the use of the rather archaic NBBY macro. :-)